controllers/C_Prescription.class.php

```diff
@@ -622,7 +622,11 @@ function multiprint_action($id = "") {
 
 		$this->multiprint_footer($pdf);
 
-		$pdf->ezStream();
+			$pFirstName = $p->patient->fname; //modified by epsdky for prescription title change to include patient name and ID
+			$pFName = preg_replace("/[^A-Za-z]/", '', $pFirstName);
```

Comment:
Recommend using the following function for this instead:
convert_safe_file_dir_name()
(this function can be used anywhere)
https://github.com/openemr/openemr/blob/master/library/sanitize.inc.php#L32